airflow-core/src/airflow/ui/src/i18n/config.ts

```diff
@@ -22,23 +22,23 @@ import Backend from "i18next-http-backend";
 import { initReactI18next } from "react-i18next";
 
 export const supportedLanguages = [
-  { code: "en", name: "English" },
-  { code: "ar", name: "العربية" },
-  { code: "ca", name: "Català" },
-  { code: "de", name: "Deutsch" },
-  { code: "es", name: "Español" },
-  { code: "fr", name: "Français" },
-  { code: "he", name: "עברית" },
-  { code: "hi", name: "हिन्दी" },
-  { code: "hu", name: "Magyar" },
-  { code: "it", name: "Italiano" },
-  { code: "ko", name: "한국어" },
-  { code: "nl", name: "Nederlands" },
-  { code: "pl", name: "Polski" },
-  { code: "pt", name: "Português" },
-  { code: "tr", name: "Türkçe" },
-  { code: "zh-CN", name: "简体中文" },
-  { code: "zh-TW", name: "繁體中文" },
+  { code: "en", name: "English", separator: "," },
```

Comment:
Just a nit - is "separator" the right term for "thousand separator"? I first thought about this would also be the term for the separation between digits and fractions as well. But not being native speaker.

Reply:
I've finally figured out how to do it using only the locale name, so this definition is not needed anymore